rails_event_store_active_record/lib/rails_event_store_active_record/generators/templates/no_global_stream_entries_template.rb

```diff
@@ -0,0 +1,48 @@
+class NoGlobalStreamEntries < ActiveRecord::Migration<%= migration_version %>
+  def change
+    case ActiveRecord::Base.connection.adapter_name
+    when "SQLite"
+      rename_table :event_store_events, :old_event_store_events
+      create_table(:event_store_events, force: false) do |t|
+        t.references  :event,       null: false, type: :string, limit: 36
+        t.string      :event_type,  null: false
+        t.binary      :metadata
+        t.binary      :data,        null: false
+        t.datetime    :created_at,  null: false
+      end
+      add_index :event_store_events, :event_id, unique: true
+      add_index :event_store_events, :created_at
+      add_index :event_store_events, :event_type
+
+      execute <<-SQL
+        INSERT INTO event_store_events(event_id, event_type, metadata, data, created_at)
+        SELECT id, event_type, metadata, data, created_at FROM old_event_store_events;
+      SQL
+      drop_table :old_event_store_events
+    when "PostgreSQL"
+      rename_column :event_store_events, :id, :event_id
+      change_column_default :event_store_events, :event_id, nil
+      add_column :event_store_events, :id, :serial
+
+      execute <<-SQL
+        ALTER TABLE event_store_events DROP CONSTRAINT event_store_events_pkey;
+        ALTER TABLE event_store_events ADD PRIMARY KEY (id);
```

Comment:
` ALTER TABLE event_store_events ADD PRIMARY KEY (id)` does that indicate auto-increment?

Reply:
or maybe that `add_column :event_store_events, :id, :serial` ? :)

Reply:
It does via `serial`: https://www.postgresql.org/docs/8.1/datatype.html#DATATYPE-SERIAL